### PR TITLE
Null Timestamp

### PR DESCRIPTION
**Pull request by LauraLaureus**
Refered issue #100

### CODE REVIEW

File: Signal-Windows/Controls/ConversationListElement.xaml.cs [hunk 1]
```diff
@@ -109,7 +109,7 @@ public string Initials
         private string _LastMessageTimestamp = string.Empty;
         public string LastMessageTimestamp
         {
-            get { return _LastMessageTimestamp; }
+            get { return _LastMessageTimestamp.Contains("1970") ? string.Empty : _LastMessageTimestamp; }
```
Comment (Trolldemorted) [inline Signal-Windows/Controls/ConversationListElement.xaml.cs]: This works, but it is a bit hackish. We should not store a different string in _LastMessageTimestamp than we are displaying. Maybe return string.empty in `Utils.GetTimestamp`?

**Comment by LauraLaureus** [comment]
@SERVCUBED  only Utils.GetTimestamp(long timestamp) or you need Utils.GetTimestamp(DataTime timestamp) too?

**Comment by SERVCUBED** [comment]
The problem is when the numerical timestamp value passed into `Utils.GetTimestamp(long timestamp)` is zero. Only that one needs to be changed as it then calls `Utils.GetTimestamp(DataTime timestamp)`.

**Comment by LauraLaureus** [comment]
Take a look!

**Comment by Trolldemorted** [comment]
Please squash the two commits into one (since the first is reverted by the second) and we are good to go!

**Comment by Trolldemorted** [comment]
Now you unfortunately pushed 4 commits.

You are only changing 2 lines of code, so to keep the commit history clean (and accountable) please change this PR so it contains only one commit.

You can do either:
- remove the last 4 commits locally (`git reset --soft HEAD~4`), add&commit again, and force-push (`git push -f`)
or
- delete the local branch, create it again from the current master, add your code again, commit it, and force-push (`git push -f`)

**Comment by LauraLaureus** [comment]
xD Sourcetree gui made another commit for the squash. 
I used the commands you gave me to fix it. Thank you 💯

**Comment by Trolldemorted** [comment]
I am really very sorry to bother you again, but your branch is now 4 commits behind our master - did you perhaps try *both* ways? It looks like you went back to the master *and* (afterwards) discarded 4 commits, so your PR now contains your changes and the ones that happenend in those 4 commits you removed.

A side note: If you would have had created a seperate branch (anything but master) before committing, git would know that an explicit branch away from master happened and github would automagically resolve this and replay your changes on the top of our master. I'd recommend to never use the master branch for PRs, a new branch ist just `git checkout -b branchname` away.

**Comment by LauraLaureus** [comment]
I think that difference of commits is that i forked the project before them [I forked 3 threes ago and your last commits was 2 days ago]. 
I'm trying to figure out how to take thos commits and pull you the changes as you wanted them.

**Comment by Trolldemorted** [comment]
I think it could be easiest if you'd clone the repo again, branch off from master with `git checkout -b branchname`, make your changes in that branch, push it to your repository, and create a PR from the new branch to our master.

Even if our master changes in the meanwhile we will be able to apply your commit on top of it.

**Comment by Trolldemorted** [comment]
superseded by #103

**Pull request closed without merging**